main.py

```diff
@@ -0,0 +1,62 @@
+"""Main module contains only the main() function 
+which combines all of the other modules to create the game play."""
+from os import system
+from board import Board
+from Team import Codegiver, Decoder
+from win import win_condition
+
+
+def main():
+    """main function contains entire game play"""
+    # This section creates the board and the blue and red team codegiver.
+    board = Board()
+    red_codegiver = Codegiver("red")
+    blue_codegiver = Codegiver("blue")
+
+    print(board.output(2))
+    input('\nPress "Enter" when you are ready to begin the game: ')
+    while True:
+        # this section clears the terminal and then ask the codegiver to give their word
+        system("clear")
+        input("\nIt is now the Codegiver turn.\nWARNING: all word guessers must look away from the screen!\n\nPress \"Enter\" when you are ready: ")
+        system("clear")
+        print(board.output(1))
+        if board.up_next == "red":
+            red_codegiver.create_code_word()
+        else:
+            blue_codegiver.create_code_word()
+
+        system("clear")
+        input('\nPress "Enter" to continue to the guesser\'s turn: ')
+        system("clear")
+        # Ask for a guess from the red decoders
+        if board.up_next == "red":

```

Comment:
There is a lot of duplicated logic in this if/else. Could you do something like this?

```python
codegiver = red_codegiver if board.up_next == "red" else blue_codegiver
codegiver.create_code_word()
decoder = Decoder(codegiver, board, board.up_next)
...
decoder.take_guess()
...
win_condition(decoder, board)
...
```